src/utility/replace_var.c

```diff
@@ -0,0 +1,96 @@
+/*
+** EPITECH PROJECT, 2020
+** get_var.c
+** File description:
+** get_var
+*/
+
+#include <shell.h>
+#include <stdlib.h>
+#include <stdio.h>
+#include "utility.h"
+#include "shell.h"
+#include <string.h>
+
+char * replace_var(char *cmd, env_t *env)
+{
+    char *isolated_var = NULL;
+    char *value = NULL;
+    char *new_cmd = NULL;
+    int var_len = 0;
+
+    if (cmd[0] == '\'')
+        return (cmd);
+    isolated_var = isolate_var(cmd, env);
```

Comment:
Your var handling will only work if there is one or less variables on a single string. Ex: "$VAR$VAR2" will parse $VAR but not $VAR2